applications/CompressiblePotentialFlowApplication/CMakeLists.txt

```diff
@@ -20,17 +20,18 @@ set( KRATOS_COMPRESSIBLE_POTENTIAL_APPLICATION_CORE
     ${CMAKE_CURRENT_SOURCE_DIR}/compressible_potential_flow_application.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/compressible_potential_flow_application_variables.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/incompressible_potential_flow_element.cpp
+    ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/compressible_potential_flow_element.cpp
     ${CMAKE_CURRENT_SOURCE_DIR}/custom_elements/incompressible_adjoint_potential_flow_element.cpp
```

Comment:
It's a good practice to sort includes and sources in alphabetic order.

Reply:
Thanks for taking the time to review and all of your comments Ruben! Going through one by one